ami/exports/tests.py

```
@@ -302,3 +303,186 @@ def test_non_member_cannot_create_export(self):
+        )
+        file_url = data_export.run_export()
+        self.assertIsNotNone(file_url)
+        file_path = file_url.replace("/media/", "")
+        with default_storage.open(file_path, "r") as f:
+            rows = list(csv.DictReader(f))
+        default_storage.delete(file_path)
+        return rows
+
+    def test_ml_prediction_only(self):
+        """Occurrence with only ML prediction: machine prediction fields populated, verified_by null."""
+        occurrence, classification = self._create_occurrence_with_prediction()
```

Ruff (0.15.9)

[warning] Unpacked variable `classification` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)